configs/auto-sgen-aotjit-serial-binary.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "auto-sgen-aotjit-serial-binary",
+  "Count": 10,
+  "Mono": "$ROOT/bin/mono-sgen",
+  "AOTOptions": [
+    "-O=all",
```

Comment:
I assumed we would pass `-O=all` on iOS, but apparently we just use the default optimization set. I'll adjust that.